website/static/js/components/legend.js

```diff
@@ -1,17 +1,56 @@
 require('css/legend.css');
 var m = require('mithril');
 
+/** Finds the index of a string in an array
```

Comment:
Actually neither are, I was working under the assumption that everything in the legend was also in `CATEGORY_MAP`, my mistake.

`find_index` isn't because it's an array of objects, so you can't easily use something like `.indexOf` to query on an attribute of each object. You *could* do some magic with `jQuery.grep`, or, more simply modify the sorting function we're using to something that special cases what we want to push to the back, like 
```js
data.sort(function (a, b) {
    if (a.label === 'Other' && b.label !== 'Uncategorized') {
        return 1;
    }
    if (b.label === 'Other' && a.label !== 'Uncategorized') {
        return -1;
    }
    return a.label.localeCompare(b.label);
});
```